.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: required
+sudo: false
```

Comment:
the default is based on when the repo was created so i'd rather be explicit about it.